web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -206,7 +249,8 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   openPlanRepShardDialog() {
-    this.dialogSettings = new DialogSettings('Reparent', `Planned Reparent in Shard ${this.keyspaceName}/${this.shardName}`, '',
+    this.dialogSettings = new DialogSettings('Reparent', `Planned Reparent in Shard ${this.keyspaceName}/${this.shardName}`,
+                                             'Designate a new master. Assumes the old master is still healthy and reachable.',
```

Comment:
How about "Failover to a new master"?

That's a much strong language and better implies the consequences than "designate". "designate" sounds like you're marking a tablet as new master, but you're not executing on it and that would be wrong?

Same comment for emergency reparent.